docs/src/use/configure/ignore.md

```diff
@@ -34,6 +34,33 @@ export default defineConfig([globalIgnores([".config/*"])]);
 
 This configuration specifies that all of the files in the `.config` directory should be ignored. This pattern is added after the default patterns, which are `["**/node_modules/", ".git/"]`.
 
+By default, ESLint lints files that match the patterns `**/*.js`, `**/*.cjs`, and `**/*.mjs`. You can ignore them with `globalIgnores()` if you don't want ESLint to lint these files:
+
+```js
+// eslint.config.js
+import { defineConfig, globalIgnores } from "eslint/config";
+
+export default defineConfig([
+	globalIgnores(["**/*.js", "**/*.cjs", "**/*.mjs"]),
+]);
+```
+
+When non-JS files are specified in the `files` property, ESLint still lints files that match the default patterns. To lint only the files specified in the `files` property, you must ignore the default file patterns:
+
+```js
+// eslint.config.js
+import { defineConfig, globalIgnores } from "eslint/config";
+
+export default defineConfig([
+	globalIgnores(["**/*.js", "**/*.cjs", "**/*.mjs"]),
+	{
+      files: ["vite.config.ts"],
+	  ignores: ["./dist/**"]
+      rules: { ... }
```

Comment:
```suggestion
		files: ["**/*.ts"],
		ignores: ["dist/**"],
		rules: { /* ... */ },
```

To fix indentation, fix syntax, and make the `ignores` make sense with the specified `files`.